### PR TITLE
fix add asset form

### PR DESCRIPTION
**Pull request by akash1810**
Previously, you had to refresh the page after adding an asset because we weren't updating state.

This change also shows the asset list iff there isn't a current asset.

add-asset

### CODE REVIEW

**Comment by akash1810** [comment]
cc @blishen

File: public/video-ui/src/actions/VideoActions/createAsset.js [hunk 1]
```diff
@@ -2,14 +2,14 @@ import { browserHistory } from 'react-router';
...
 const BLANK_ASSET = {
-  uri: '',
-  mimetype: '',
-  version: 0
+  uri: ''
 };
 
-function requestAssetCreate() {
+function requestAssetCreate(video) {
+  debugger;
```
Comment (mbarton) [inline public/video-ui/src/actions/VideoActions/createAsset.js]: 😱

File: public/video-ui/src/components/VideoAssets/VideoAssets.js [hunk 2]
```diff
@@ -121,7 +117,7 @@ class VideoAssets extends React.Component {
           {this.renderAssetEdit()}
           {this.renderList()}
           {!this.props.video.assets.length ? <span>No assets found</span> : false}
-          {!this.state.showAssetList && this.props.video.assets.length ? <button className="video-assets__show-btn" type="button" onClick={this.showAssetList}>Show all assets</button> : false}
+          {!this.state.showAssetList && this.props.video.assets.length && !!this.props.video.activeVersion ? <button className="video-assets__show-btn" type="button" onClick={this.showAssetList}>Show all assets</button> : false}
```
Comment (mbarton) [inline public/video-ui/src/components/VideoAssets/VideoAssets.js]: I think this would be clearer refactored out of a one-liner?

**Comment by akash1810** [comment]
@mbarton good to go?

**Pull request merged**